docs/examples/nested_regions.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region, Page
+from selenium.webdriver.common.by import By
+
+
+class MainPage(Page):
+
+    @property
+    def menu1(self):
+        root = self.find_element(By.ID, "menu1")
+        return Menu(self, root=root)
+
+    @property
+    def menu2(self):
+        root = self.find_element(By.ID, "menu2")
+        return Menu(self, root=root)
+
+
+class Menu(Region):
+    _root_locator = (By.CLASS_NAME, 'menu')
```

Comment:
You're passing the root element when instantiating these regions, so `_root_locator` is unecessary, and unused.